ws-server/src/server.rs

```diff
@@ -253,45 +254,12 @@ async fn handshake<M: Middleware>(socket: tokio::net::TcpStream, mode: Handshake
 		}
 		HandshakeResponse::Accept { conn_id, methods, resources, cfg, stop_monitor, middleware, id_provider } => {
 			tracing::debug!("Accepting new connection: {}", conn_id);
-			let key = {
-				let req = server.receive_request().await?;
```

Comment:
factored to this into a helper function because once this returns `Err` it's just dropped without sending a proper HTTP response.

so every thing that returns an error we just send `HTTP status code 403`